### ISSUE TITLE
bash completion

### ISSUE THREAD

**Issue by kindlychung** [LoopPerfect/buckaroo#182]
would be nice to have autocompletion on the commandline.

**Comment by njlr** [comment]
Agreed! I'm not sure where this will fit in the list of priorities, but this would definitely be nice to have. 👍

**Comment by njlr** [comment]
Will endeavour to include this in first release of Buckaroo Redux. Likely we can install using the `.deb` file. 

Basically this file (https://github.com/LoopPerfect/buckaroo/blob/buckaroo-redux/buckaroo_bash_completion.bash) needs to be copied into `/etc/bash_completion.d/`.
